app/messages.py

```diff
@@ -9,6 +12,9 @@
 FIELD_AVAILABLE_TO_MENTOR_IS_INVALID = {
     "message": "Field available_to_mentor" " is not valid."
 }
+PASSWORD_INPUT_BY_USER_HAS_INVALID_LENGTH = {
+    "message": get_length_validation_error_message("password", PASSWORD_MIN_LENGTH, PASSWORD_MAX_LENGTH)
+}
```

Comment:
May be you can directly write that message rather than using the method as this may seem something different from other messages .  Although this should work fine but still i think using a direct message will be a lot easier then using a method .? 🤔

Reply:
I wanted to avoid hard coding the message. But yeah, the direct message is much easier and neater. I've updated the changes

Reply:
@Vuyanzi I understand your point. And in another PR, someone used the hardcoded message approach and I wanted the approach you did. But I am not a fan of either of those. Because the hardcoded approach, allows for internationalization (having different languages supported)... but it does not allow for parameterization, in terms of receiving the variable instead of having the limit hardcoded. Maybe you can do it hardcoded for now, but we have to refactor this somehow